src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/Tool/AppProvisioningTool.cs

```diff
@@ -82,11 +82,16 @@ public AppProvisioningTool(string commandName, ProvisioningToolOptions provision
 
             if (projectDescription == null)
             {
-                ConsoleLogger.LogMessage($"No project found in {ProvisioningToolOptions.ProjectPath}.", LogMessageType.Error);
+                ConsoleLogger.LogMessage(string.Format(Resources.NoProjectFound, ProvisioningToolOptions.ProjectPath), LogMessageType.Error);
             }
             else
             {
-                ConsoleLogger.LogMessage($"Detected project type {projectDescription.Identifier}.");
+                ConsoleLogger.LogMessage(string.Format(Resources.DetectedProjectType, projectDescription.Identifier));
+                if (!string.IsNullOrEmpty(projectDescription.Identifier))
+                {
+                    string projectType = Regex.Replace(projectDescription.Identifier, "dotnet-", "");
```

Comment:
>Replace [](http://example.com/codeflow?start=47&length=7)

do you need regers replace here or string replace would be enough?

Reply:
fixed and moved https://github.com/dotnet/Scaffolding/pull/1593